opentech/apply/funds/serializers.py

```diff
@@ -80,11 +101,12 @@ def get_questions(self, obj):
 
 
 class SubmissionActionSerializer(serializers.ModelSerializer):
-    actions = ActionSerializer(source='*')
+    actions = ActionSerializer(source='*', read_only=True)
+    action = serializers.CharField(write_only=True)
```

Comment:
Seeing as we don't use the proper create method is there any value to doing this? I would hate for someone to start posting actions and easily by passing the view. (always possible but lets not make it easy)